### PR TITLE
Add 4_3E

### PR DESCRIPTION
**Pull request by Apuru**
I would add this under "Cores & Other Farming" on the wiki.

Requires 2 echelons.
Echelon 1, level 50+ frontal F formation setup
Echelon 2 dummy.

Good for leveling 50-80

### CODE REVIEW

File: src/main/kotlin/com/waicool20/wai2k/script/modules/combat/maps/Map4_3E.kt [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        planPath()
+        waitForTurnEnd(4)
+        handleBattleResults()
+    }
+
+    private suspend fun planPath() {
+        logger.info("Entering planning mode")
+        mapRunnerRegions.planningMode.click(); yield()
+
+        logger.info("Selecting ${nodes[0]}")
```
Comment (waicool20) [inline src/main/kotlin/com/waicool20/wai2k/script/modules/combat/maps/Map4_3E.kt:65]: Wrong node

**Review by waicool20 — CHANGES_REQUESTED** [review]
- Image has that emulator tap thing, zoom-anchor.png can be deleted. 
- Probably best to crop to the region size defined in `MapRunnerRegions.window` for best matching results
- Wrong node in one of the logging statements
- Next time update your project by forwarding your branch and maybe squashing relevant commits.

**Comment by waicool20** [comment]
Oh and also put your stuff in its own feature branch next time <3

File: src/main/kotlin/com/waicool20/wai2k/script/modules/combat/maps/Map4_3E.kt [hunk 2]
```diff
@@ -63,7 +63,7 @@ class Map4_3E(scriptComponent: ScriptComponent) : HomographyMapRunner(scriptComp
         mapRunnerRegions.planningMode.click(); yield()
 
         logger.info("Selecting ${nodes[0]}")
-        nodes[1].findRegion().click()
+        nodes[0].findRegion().click()
```
Comment (waicool20) [inline src/main/kotlin/com/waicool20/wai2k/script/modules/combat/maps/Map4_3E.kt:66]: You sure you want to be clicking node 0?

Comment (Apuru) [reply src/main/kotlin/com/waicool20/wai2k/script/modules/combat/maps/Map4_3E.kt:66]: Yup should be heliport node.

image

I still need to crop the image so it fits the size and redo coords

Comment (waicool20) [reply src/main/kotlin/com/waicool20/wai2k/script/modules/combat/maps/Map4_3E.kt:66]: Might as well do it in another branch and resubmit a new PR then

**Pull request closed without merging**